kolibri/plugins/style_guide/assets/src/views/content/button.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+
+  <div>
+    Button
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+  };
+
+</script>
+
+
+<style lang="stylus" scoped>
+
+  @require '~kolibri.styles.definitions'
```

Comment:
we probably shouldn't require the core style definitions in `style_guide` - I would expect this to only be used inside the individual core components